internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -3029,6 +3041,10 @@ func flattenKubernetesClusterNetworkProfile(profile *managedclusters.ContainerSe
 			lb["outbound_ports_allocated"] = v
 		}
 
+		if v := lbp.EnableMultipleStandardLoadBalancers; v != nil {
+			lb["multiple_standard_load_balancer_enabled"] = v
```

Comment:
this should be plural? as the property is + it doesn't make sense without a s?
```suggestion
		if v := lbp.EnableMultipleStandardLoadBalancers; v != nil {
			lb["multiple_standard_load_balancers_enabled"] = v
```